src/cf-modals/usage.md

```diff
@@ -0,0 +1,175 @@
+Modals are components that can be shown and hidden as a popup overlay.
+
+[`cf-core`](../cf-core) and [`cf-icons`](../cf-icons) components are
+dependencies of this component.
+
+> NOTE: If you use `cf-modals.less` directly,
+  be sure to run the file through
+  [Autoprefixer](https://github.com/postcss/autoprefixer),
+  or your compiled Capital Framework CSS will
+  not work perfectly in older browsers.
+
+
+## Table of contents
+
+- [Variables](#variables)
+  - [Color variables](#color-variables)
+- [Recommended modal patterns](#recommended-modal-patterns)
+  - [Dialog modal](#dialog-modal)
+  - [Alert modal](#alert-modal)
+  - [Visible modifier](#visible-modifier)
+
+## Variables
+
+Theme variables for setting the color and sizes throughout the project.
+Overwrite them in your own project by duplicating the variable `@key: value`.
+
+### Color variables
+
+```
+@modal-backdrop:   @black;
+@modal-body:       @white;
+@modal-footer:     @gray-5;
+@modal-border-top: green;
+```
+
+
+## Recommended modal patterns
+
+### Dialog modal
+
+A regular dialog modal does not contain a call to action for the user and
+can simply be closed without further interaction.
+
+<div class="o-modal"
+     aria-hidden="true"
+     role="dialog"
+     aria-labelledby="example-modal-1-title"
+     aria-describedby="example-modal-1-desc">
+    <div class="o-modal_backdrop"></div>
+    <div class="o-modal_container">
```

Comment:
Could you eliminate the container?

Reply:
I didn't find a way. I just realized this essentially rebuilt the `o-modal_offset` that I removed earlier, lol. It's needed for properly handling the modal dimensions and the area that can be clicked to close the modal.